deep_qa/layers/attention/attention.py

```diff
@@ -8,18 +8,22 @@
 
 class Attention(Layer):
     '''
-    This Layer takes two inputs: a vector and a matrix.  We compute the similarity between the
-    vector and each row in the matrix, and then perform a softmax over rows using those computed
-    similarities.  We handle masking properly for masked rows in the matrix, though we ignore any
-    masking on the vector.
+    This Layer takes two inputs: a vector and a matrix.  We compute the
+    similarity between the vector and each row in the matrix, and then perform
+    a softmax over rows using those computed similarities.  We handle masking
+    properly for masked rows in the matrix, though we ignore any masking on the vector.
 
-    By default similarity is computed with a dot product, but you can alternatively use a
-    parameterized similarity function if you wish.
+    By default similarity is computed with a dot product, but you can
+    alternatively use a parameterized similarity function if you wish.
 
     Input shapes:
```

Comment:
I'd probably change this to match the `Input`, `Output`, and `Parameters` style that we used in other places.  Also, the similarity function params need to be changed to match what changed in `MatrixAttention`.  This layer will not load correctly.  We should probably open an issue for this instead of doing it in this PR, though.